packages/cli/oclif.manifest.json

```diff
@@ -776,6 +776,37 @@
       },
       "args": {}
     },
+    "hydrogen:env:push__unstable": {
+      "id": "hydrogen:env:push__unstable",
+      "description": "Push your local variables from .env to your Hydrogen storefront.",
```

Comment:
you need to regenerate your oclif file - run `npm run build` from `packages/cli`

Reply:
Yup, forgot to run after merging Graham's updates!